crates/backend/Cargo.toml

```diff
@@ -12,7 +12,9 @@ actix-files = "0.5.0"
 clap = "3.0.0-beta.4"
 serde_json = "1.0.68"
 serde = "1.0.130"
-ts-rs = {version = "4.0.0", features = ["serde-compat"] }  
+ts-rs = { git = "https://github.com/Aleph-Alpha/ts-rs", branch = "main", features = [
```

Comment:
We will replace the github URL with the next version once released.
Note: backend still needs this dependency. Not for the actual type generation. Rather because ts-rs's derive macro that we need to wrap types with.